main.js

```diff
@@ -6,6 +6,88 @@ const logo = document.querySelector('#logo-placeholder')
 const menuElements = document.querySelector('#nav-list')
 const closeIcon = document.querySelector('#close-Icon')
 const body = document.getElementsByTagName('body')[0]
+const worksSection = document.querySelector('#works');
+
```

Comment:
- [ ] You have a linter diable rule on the first line of this file. According to Microverse, you are not meant to use any linters disable rule unless it was stated. I'll suggest you remove this and try to find the best way to fix this as linters are meant to enforce the best practices. :+1: :smile:

Reply:
@V-Blaze Yes, I totally forgot it. Thanks a lot. I fixed it immediately.